internal/generate/generate.go

```diff
@@ -278,7 +282,7 @@ func (c *CA) StoreAsSecret(ctx context.Context, k8sClient *kubernetes.Clientset)
 
 	k8sSecrets := k8sClient.CoreV1().Secrets(c.SecretNamespace)
 	_, err := k8sSecrets.Create(ctx, secret, meta_v1.CreateOptions{})
-	if k8sErrors.IsAlreadyExists(err) {
+	if k8sErrors.IsAlreadyExists(err) && force {
```

Comment:
in case of the secret is already exists and force is true, I just take simple approach to throw error, instead of fetching existing value, and update ca struct c here.